receiver/gitproviderreceiver/internal/scraper/githubscraper/genqlient.graphql

```diff
@@ -36,6 +36,33 @@ query getRepoDataBySearch(
   }
 }
 
+query getRepoDataByTeam(
+    $org: String!
+    $team: String!
+    # @genqlient(pointer: true)
+    $repoCursor: String
+){
+	organization(login: $org){
```

Comment:
Do you know if `$org` in this case can actually be a user?

Reply:
it can in fact be a user